src/python/grapl_analyzerlib/grapl_analyzerlib/comparators.py

```diff
@@ -199,9 +199,9 @@ def __init__(self, predicate: str, value: StrOrNot, distance: int):
 
     def to_filter(self) -> str:
```

Comment:
I think this is a misfire. We should add a `self.distance = distance` to the constructor. The distance parameter is a requirement for this filter.

Reply:
good call